tools/git_lfs.rst

```diff
@@ -31,20 +36,70 @@ Once ``git-lfs`` is installed, run
    git config --global lfs.batch false
    git lfs install
 
-to install the necessary hooks in your :file:`~/.gitconfig` file.
+to configure Git to use Git LFS in your :file:`~/.gitconfig` file.
+
+Next, we need to tell Git that DM's LFS storage services don't need a password for reading.
+Begin by activating Git's `credential store`_ helper for DM's Git LFS storage services by adding these lines into your :file:`~/.gitconfig` file:
+
+.. code-block:: text
+
+   [credential "https://lsst-sqre-prod-git-lfs.s3-us-west-2.amazonaws.com"]
+           helper = store
+   [credential "https://s3.lsst.codes"]
+           helper = store
+
+Then add DM's Git LFS storage servers to your :file:`~/.git-credentials` file:
+
+.. code-block:: text
+   
+   https://:@lsst-sqre-prod-git-lfs.s3-us-west-2.amazonaws.com
+   https://:@s3.lsst.codes
+
+*Notice that the username is purposefully missing from these lines; this tells the Git credential store that you're opting for anonymous access.*
+
+Next, decide whether you will need to push Git LFS data, or only clone and pull form Git LFS managed repositories.
+This affects how you set up authentication to DM's Git LFS server (we only configured anonymous access to the *storage services* so far).
```

Comment:
"we have only configured" or "we only configure"

"storage services" suggests to me that you are only configuring anonymous access for pushes and not pulls. By "storage services" do you mean "services for retrieving data" rather than "services for storing data"?